src/custom/components/Button/index.tsx

```diff
@@ -254,12 +254,13 @@ export function ButtonDropdownLight({
 export const ButtonStar = ({
   fill = false,
   size = '15px',
+  stroke,
   ...rest
-}: { fill: boolean; size?: string } & HTMLAttributes<HTMLDivElement>) => {
+}: { fill: string | false; size?: string; stroke: string } & HTMLAttributes<HTMLDivElement>) => {
```

Comment:
if fill is false it'll always default to 'transparent' right? just set it as a string and pass 'transparent' as the default prop in the destructure